src/GeositeFramework/js/widgets/map_utils/measure/AgsMeasure.js

```diff
@@ -261,10 +284,21 @@ define([
 
                 // Calculate the length of the line, using a geographic coordindate system
                 geographicLine = webMercatorUtils.webMercatorToGeographic(line);
-                var geoLineLength = geodesicUtils.geodesicLengths([geographicLine], options.esriLengthUnits)[0];
+                var geoLineLength = getLength(geographicLine);
 
                 // Format the segment and line lengths
-                tipText = formatTooltip(geoLineLength, _renderedLength + geoLineLength);
+                var totalLength = _.map(
```

Comment:
I refactored this function bit by splitting the functions and values into separate pieces to make the sequence a bit clearer.